pkgs/development/tools/ilspycmd/default.nix

```diff
@@ -1,52 +1,59 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, buildDotnetModule
-, dotnetCorePackages
-, powershell
-, autoSignDarwinBinariesHook
-, glibcLocales
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  powershell,
+  autoSignDarwinBinariesHook,
+  glibcLocales,
 }:
-buildDotnetModule rec {
+buildDotnetModule (finalAttrs: rec {
```

Comment:
Did you mean to also remove `finalAttrs` here?

Reply:
No, I've kept it because the other use site seems valid.